### ISSUE TITLE
admin-views raises RemovedInDjango110Warning warning

### ISSUE THREAD

**Issue by dbinoj** [frankwiles/django-admin-views#30]
admin-views in Django 1.9.7 raises the following message during dev server startup system checks.

```
/home/vagrant/.Env/apps/local/lib/python2.7/site-packages/admin_views/admin.py:36: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  view=self.admin_site.admin_view(view_func)
```

**Comment by dbinoj** [comment]
Someone please fix/merge the fix as Django 1.10 is out now.
I'm currently using `pip install git+https://github.com/dbinoj/django-admin-views.git@patch-1`

**Comment by frankwiles** [comment]
I just released version 0.8.0 which has this fixed in it.

**Comment by dbinoj** [comment]
Thanks!
